extension/goplugin/environment_test.go

```diff
@@ -50,6 +50,16 @@ var _ = Describe("Environment", func() {
 		env = goplugin.NewEnvironment("test", nil, nil)
 	})
 
+	Describe("Internal state consistency", func() {
```

Comment:
still failes if Schemas() returns empty list:
```
13:43:22.906 extest CRITICAL  failed to load schemas extensions: failed to init db: No schemas in Manager. Did you remember to load them?
```

Reply:
it should be already fixed